website/ember_osf_web/views.py

```diff
@@ -15,5 +17,10 @@
 def use_ember_app(**kwargs):
     if PROXY_EMBER_APPS:
         resp = requests.get(EXTERNAL_EMBER_APPS['ember_osf_web']['server'], stream=True, timeout=EXTERNAL_EMBER_SERVER_TIMEOUT)
-        return Response(stream_with_context(resp.iter_content()), resp.status_code)
-    return send_from_directory(ember_osf_web_dir, 'index.html')
+        resp = Response(stream_with_context(resp.iter_content()), resp.status_code)
+    else:
+        resp = send_from_directory(ember_osf_web_dir, 'index.html')
+    if session.data.get('status'):
+        status = [{'id': stat[5] if stat[5] else stat[0], 'class': stat[2], 'jumbo': stat[1], 'dismiss': stat[3], 'extra': stat[6]} for stat in session.data['status']]
```

Comment:
So we're not passing in the message here, you're just passing in the message id which corresponds to something ember-side?  Might want to pass in whatever we use osf-side for consistency.

Reply:
Drive-by: Isn't `stat` a namedtuple? If so, it'd be more readable to do `stat.id`, `stat.class`, etc.

Reply:
Not at this point. Stat is a list when it gets here

Reply:
Ah ok; makes sense.